src/Symfony/Component/Security/Core/User/UserInterface.php

```diff
@@ -19,10 +19,10 @@
  * password (for checking against a submitted password), assigning roles
  * and so on.
  *
- * Regardless of how your user are loaded or where they come from (a database,
- * configuration, web service, etc), you will have a class that implements
- * this interface. Objects that implement this interface are created and
- * loaded by different objects that implement UserProviderInterface
+ * Regardless of how your users are loaded or where they come from (a database,
+ * configuration, web service, etc.), you will have a class that implements
+ * this interface. Objects of that class are created and loaded by different
+ * objects that implement UserProviderInterface
```

Comment:
classes that ....

?

Reply:
I would only fix `user` and `etc`, the rest does not sound better than what we have already TBH

Reply:
Ok, thank you, I changed it back. For me it sounds strange to say that an _object_ implements an interface.

Reply:
@OskarStark I am not sure how to understand your review..